openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/workflows/dbt.md

```diff
@@ -104,6 +104,8 @@ In case of multiple projects in a dbt cloud account, specify the project's ID fr
 
 If left empty, the dbt artifacts will be fetched from the most recent run on dbt cloud.
 
+To find your project ID, sign in to your dbt cloud account and choose a specific project. Take note of the url which will be similar to `https://cloud.getdbt.com/#/accounts/1234/settings/projects/6789/`, the project ID is `6789`.
+
```

Comment:
We can make this more generic for the users to understand here..
something like `project/<projectid>` and then give an example `project/1234`

Reply:
Keeping it a numeric value instead of a generic value was on purpose since users were getting confused already what to put there.
Directly adding an example makes things more clear.